.github/workflows/linter.yaml

```diff
@@ -7,16 +7,14 @@ on:
   push:
     branches: [ main ]
   pull_request:
-    branches: [ main ]
+    branches-ignore: [ production ]
```

Comment:
Don't we want to run tests on prod?

Reply:
but everything on prod should have come from main, so we don't need it to run again